### PR TITLE
Editorial nits

### PR DESCRIPTION
**Pull request by timbray**
Signed-off-by: Tim Bray <tbray@textuality.com>

Nothing fancy here, just like the title says.

### CODE REVIEW

File: draft-ietf-jsonpath-base.md [hunk 1]
```diff
@@ -550,7 +550,9 @@ The syntax and semantics of segments are defined in {{segments-details}}.
 
 In this document, the semantics of a JSONPath query define the
 required results and do not prescribe the internal workings of an
-implementation.
+implementation.  This document may describe semantics in a procedural
+step-by-stap fashion, but such descriptions are non-normative and do not
```
Comment (cabo) [inline draft-ietf-jsonpath-base.md:554]: Well, the descriptions are meant to be normative, but the way they are described is not the only way of implementing them.

**Review by cabo — APPROVED** [review]
Thank you.
The text about how the descriptions are not prescribing implementation details but are still normatively describing the semantics could be improved, e.g., using some "same result" language.

File: draft-ietf-jsonpath-base.md (hunk 1 above)
Comment (timbray) [reply draft-ietf-jsonpath-base.md:554]: Something like "… step-by-step function, which are normative in the sense that any implementation MUST produce an identical result, but not in the sense that implementors are required to use the same algorithms" ?

Comment (cabo) [inline draft-ietf-jsonpath-base.md:554]:
```suggestion
step-by-step fashion, but such descriptions are normative only in the sense that any implementation MUST produce an identical result, but not in the sense that implementors are required to use the same algorithms.
```

File: draft-ietf-jsonpath-base.md [hunk 2]
```diff
@@ -550,7 +550,9 @@ The syntax and semantics of segments are defined in {{segments-details}}.
...
+constrain implementors.
```
Comment (cabo) [inline draft-ietf-jsonpath-base.md:555]:
```suggestion

```

Comment (glyn) [reply draft-ietf-jsonpath-base.md:555]: Er, is this suggestion still needed?

Comment (glyn) [reply draft-ietf-jsonpath-base.md:555]: Oh I see.

**Review by glyn — APPROVED** [review]
LGTM, thanks.

**Pull request merged**